login.js

```diff
@@ -0,0 +1,22 @@
+const loginButton = document.querySelector(".login");
+
+// 로그인 버튼 활성화 
+function toggleButton() {
+    // 이메일, 비밀번호 유효성 검사 및 값이 있는지 확인
+    if (emailErr.textContent === "" && passwordErr.textContent === "" &&
+        emailInput.value.trim() !== "" && passwordInput.value.trim() !== "") {
+        loginButton.disabled = false;
+        loginButton.classList.add("active");
+    } else {
+        loginButton.disabled = true;
+        loginButton.classList.remove("active");
+    }
+}
```

Comment:
💊 제안
조건문이 길때는 주석도 좋지만 변수로 빼시면 가독성에 도움이 됩니다~
또한 이메일이나 비밀번호 인풋의 값이 없으면 각 인풋의 에러메시지가 있을 테니 아래 조건만 확인하셔도 될 것 같아요.

```suggestion
function toggleButton() {
    const isFormValid = emailErr.textContent === "" && passwordErr.textContent === ""; 

    loginButton.disabled = !isFormValid;
    loginButton.classList.toggle("active", isFormValid);
}
```